<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QuarkPlayer</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="quarkplayer.qrc" >
    <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>53</y>
     <width>386</width>
     <height>314</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>0</number>
    </property>
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="videoLayout" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles" >
     <property name="title" >
      <string>Recent Files</string>
     </property>
     <addaction name="separator" />
     <addaction name="actionClearRecentFiles" />
    </widget>
    <addaction name="actionAddFiles" />
    <addaction name="menuRecentFiles" />
    <addaction name="actionPlayDVD" />
    <addaction name="actionActionPlayVCD" />
    <addaction name="actionPlayURL" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuVideo" >
    <property name="title" >
     <string>Video</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAudio" >
    <property name="title" >
     <string>Audio</string>
    </property>
    <widget class="QMenu" name="menuAudioChannels" >
     <property name="title" >
      <string>Audio Channels</string>
     </property>
     <property name="icon" >
      <iconset resource="quarkplayer.qrc" >
       <normaloff>:/images/audio_channel.png</normaloff>:/images/audio_channel.png</iconset>
     </property>
     <addaction name="actionAudioChannelEmpty" />
    </widget>
    <addaction name="menuAudioChannels" />
   </widget>
   <widget class="QMenu" name="menuSubtitle" >
    <property name="title" >
     <string>Subtitle</string>
    </property>
    <widget class="QMenu" name="menuSubtitles" >
     <property name="title" >
      <string>Subtitles</string>
     </property>
     <property name="icon" >
      <iconset resource="quarkplayer.qrc" >
       <normaloff>:/images/subtitle.png</normaloff>:/images/subtitle.png</iconset>
     </property>
     <addaction name="actionSubtitleEmpty" />
    </widget>
    <addaction name="actionOpenSubtitleFile" />
    <addaction name="menuSubtitles" />
   </widget>
   <widget class="QMenu" name="menuBrowse" >
    <property name="title" >
     <string>Browse</string>
    </property>
    <widget class="QMenu" name="menuTitles" >
     <property name="title" >
      <string>Title</string>
     </property>
     <property name="icon" >
      <iconset resource="quarkplayer.qrc" >
       <normaloff>:/images/title.png</normaloff>:/images/title.png</iconset>
     </property>
     <addaction name="actionTitleEmpty" />
    </widget>
    <widget class="QMenu" name="menuChapters" >
     <property name="title" >
      <string>Chapter</string>
     </property>
     <property name="icon" >
      <iconset resource="quarkplayer.qrc" >
       <normaloff>:/images/chapter.png</normaloff>:/images/chapter.png</iconset>
     </property>
     <addaction name="actionChapterEmpty" />
    </widget>
    <widget class="QMenu" name="menuAngles" >
     <property name="title" >
      <string>Angle</string>
     </property>
     <property name="icon" >
      <iconset resource="quarkplayer.qrc" >
       <normaloff>:/images/angle.png</normaloff>:/images/angle.png</iconset>
     </property>
     <addaction name="actionAngleEmpty" />
    </widget>
    <addaction name="menuTitles" />
    <addaction name="menuChapters" />
    <addaction name="menuAngles" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQt" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>Options</string>
    </property>
    <addaction name="actionQuickSettings" />
    <addaction name="actionPreferences" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuVideo" />
   <addaction name="menuAudio" />
   <addaction name="menuSubtitle" />
   <addaction name="menuBrowse" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="mainToolBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>19</y>
     <width>386</width>
     <height>34</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>mainToolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFiles" />
   <addaction name="actionPlayDVD" />
   <addaction name="actionPlayURL" />
   <addaction name="separator" />
   <addaction name="actionPreferences" />
   <addaction name="separator" />
   <addaction name="actionExit" />
  </widget>
  <action name="actionAddFiles" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text" >
    <string>Add Files...</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/volume.png</normaloff>:/images/volume.png</iconset>
   </property>
   <property name="text" >
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/qt.png</normaloff>:/images/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionPlay" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text" >
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/fullscreen.png</normaloff>:/images/fullscreen.png</iconset>
   </property>
   <property name="text" >
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionPlayDVD" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/dvd.png</normaloff>:/images/dvd.png</iconset>
   </property>
   <property name="text" >
    <string>Play DVD</string>
   </property>
  </action>
  <action name="actionAudioChannelEmpty" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&lt;empty></string>
   </property>
  </action>
  <action name="actionSubtitleEmpty" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&lt;empty></string>
   </property>
  </action>
  <action name="actionTitleEmpty" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&lt;empty></string>
   </property>
  </action>
  <action name="actionChapterEmpty" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&lt;empty></string>
   </property>
  </action>
  <action name="actionAngleEmpty" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&lt;empty></string>
   </property>
  </action>
  <action name="actionOpenSubtitleFile" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/file_open.png</normaloff>:/images/file_open.png</iconset>
   </property>
   <property name="text" >
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClearRecentFiles" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/preference.png</normaloff>:/images/preference.png</iconset>
   </property>
   <property name="text" >
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionPlayURL" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/url.png</normaloff>:/images/url.png</iconset>
   </property>
   <property name="text" >
    <string>Play URL...</string>
   </property>
   <property name="iconText" >
    <string>Play URL...</string>
   </property>
   <property name="toolTip" >
    <string>Play URL...</string>
   </property>
  </action>
  <action name="actionActionPlayVCD" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/vcd.png</normaloff>:/images/vcd.png</iconset>
   </property>
   <property name="text" >
    <string>Play VCD</string>
   </property>
  </action>
  <action name="actionQuickSettings" >
   <property name="icon" >
    <iconset resource="quarkplayer.qrc" >
     <normaloff>:/images/preference.png</normaloff>:/images/preference.png</iconset>
   </property>
   <property name="text" >
    <string>Quick Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="quarkplayer.qrc" />
 </resources>
 <connections/>
</ui>
